tests/manual/monkey/README.md

```diff
@@ -0,0 +1,43 @@
+# Monkey Tester
+
+This is a testing tool developed to support manual ad-hoc testing of individual JavaFX controls.
+
+![screenshot](doc/screenshot.png)
+
+
+## Prerequisites
+
+JavaFX SDK is required to build the tool.  The latest SDK can be found here:
+
+https://jdk.java.net/javafx21/
+
+
+## Build
+
+The tool uses `ant` to build a non-modular JAR.  You'll need to specify the path to JavaFX SDK 20+
+(using absolute path, the script apparently does not understand ~ symbols):
+```
+ant -Djavafx.home=<JAVAFX>
```

Comment:
Try to be consistent when using parameters in code snippets. This line uses `<JAVAFX>` but on line 31 there is `/PATH`

Reply:
good point, thanks!